dev/tools/controllerbuilder/generate.sh

```diff
@@ -209,5 +209,11 @@ go run . generate-types \
     --api-version apigee.cnrm.cloud.google.com/v1alpha1 \
     --resource ApigeeInstance:GoogleCloudApigeeV1Instance
 
+go run . generate-controller \
+    --service mockgcp.cloud.apigee.v1 \
+    --api-version apigee.cnrm.cloud.google.com/v1alpha1 \
+    --kind ApigeeInstance \
+    --proto-resource GoogleCloudApigeeV1Instance
+
```

Comment:
We don't need to update this file. It is just an example (or is it a requirement now?)

Reply:
Yeah let's not add generate-controller command.